tests/integration/targets/ec2_instance/roles/ec2_instance/meta/main.yml

```diff
@@ -0,0 +1,5 @@
+dependencies:
+  - prepare_tests
+  - setup_ec2
+collections:
+  - amazon.aws
```

Comment:
I guess we should  also include `community.aws` for the other unsupported modules used here?